components/k8s-model-server/docker/Makefile

```diff
@@ -21,7 +22,23 @@ all: build
 build:
 	docker build --pull -t ${PROJECT}/model-server:${VERSION} .
 
+build-gpu:
+	docker build --pull -t ${PROJECT}/model-server:${GPU_VERSION} -f Dockerfile.gpu .
```

Comment:
version=1.0, gpu_version=1.4, then we can't tell from the tag the the image is for gpu or not.
I think we should change it to xxx/model-server-gpu:$GPU_VERSION

Reply:
Gotcha, thanks